runtime/src/bank.rs

```diff
@@ -513,40 +514,57 @@ impl Bank {
         self.genesis_creation_time + ((self.slot as u128 * self.ns_per_slot) / 1_000_000_000) as i64
     }
 
+    fn update_sysvar_account<F>(&self, pubkey: &Pubkey, updater: F)
+    where
+        F: Fn(&Option<Account>) -> Account,
+    {
+        let cold_account = self.get_cold_account(pubkey);
+        let mut new_account = updater(&cold_account);
+
+        // Normally, just use the hash from parent slot.  However, use the
+        // existing stored hash if any for the sake of bank hash's idempotent.
+        if let Some((hot_account, _)) = self.get_account_modified_since_parent(pubkey) {
```

Comment:
IMO, I just camed up this naming and like it. `new_account <=> (old_account == {hot -> cold -> frozen}_account)`. So, if this is agreeable, I'll rename `get_account_modified_since_parent` to `get_hot_account()`.

@mvines @garious Any opnitions? Mention based on https://github.com/solana-labs/solana/pull/2806/files#diff-7145777648f8c12caeb15b4e1755215cR560

Reply:
I don't find the hot/cold labeling here intuitive.   If I didn't have the additional context that a hot account is an account_modified_since_parent I don't think I would have guessed that on my own.

Reply:
Thanks for your opinion! Then, I just reverted the naming thing: https://github.com/solana-labs/solana/pull/7892/commits/4f1fe0ad467bdfbf68ada95cc7c6f5b86b6035bb